openmdao/components/tests/test_implicit_func_comp.py

```diff
@@ -98,7 +137,7 @@ def linearize(z, x, y1, y2, R_y1, R_y2, J):
 
         p_opt.model = om.ImplicitFuncComp(f, linearize=linearize)
 
-        newton = p_opt.model.nonlinear_solver = om.NewtonSolver(solve_subsystems=False, iprint=0)
+        p_opt.model.nonlinear_solver = om.NewtonSolver(solve_subsystems=False, iprint=0)
```

Comment:
Note:  if you think it worthwhile to change, there is similar code in the doc:
(openmdao/docs/openmdao_book/features/building_blocks/components/implicit_func_comp.ipynb)

Reply:
fixed